.github/workflows/deploy-vue-storefront-cloud.yml

```diff
@@ -46,64 +46,27 @@ jobs:
           NPM_REGISTRY: "https://registrynpm.storefrontcloud.io"
-
-  deploy:
+  
+  deployment-app:
     runs-on: ubuntu-latest
-    permissions:
-      deployments: write
     needs: build
     steps:
-      - uses: chrnorm/deployment-action@releases/v1
-        name: Create GitHub deployment
-        id: deployment
-        with:
-          token: "${{ github.token }}"
-          target_url: https://${{ secrets.PROJECT_NAME }}.${{ secrets.CLOUD_REGION }}.gcp.storefrontcloud.io
-          environment: production
-          initial_status: in_progress
-
-      - name: Deploy on ${{ secrets.PROJECT_NAME }}.${{ secrets.CLOUD_REGION }}.gcp.storefrontcloud.io
-        run: |
-          curl --location --request PATCH '${{ secrets.PLATFORM_API_URL || 'https://api.platform.vuestorefront.io' }}/cloud/instances/${{ secrets.PROJECT_NAME }}-${{ secrets.CLOUD_REGION }}-gcp-storefrontcloud-io/deploy' --header 'Content-Type: application/json' --data-raw '{ "cloudUserId": "${{ secrets.CLOUD_USERNAME }}", "cloudUserPassword": "${{ secrets.CLOUD_PASSWORD }}", "dockerImageHash": "${{ github.sha }}"}'
-
-      - name: Deploy middleware on ${{ secrets.PROJECT_NAME }}.${{ secrets.CLOUD_REGION }}.gcp.storefrontcloud.io/api/
-        run: |
-          if curl -f -s -H 'X-User-Id: ${{ secrets.CLOUD_USERNAME }}' -H 'X-Api-Key: ${{ secrets.CLOUD_PASSWORD }}' -H 'Content-Type: application/json' -X PATCH -d '{
-            "front_version":"${{ github.sha }}",
-            "additional_apps": {
-              "apps":[{
-                "name":"middleware",
-                "tag":"${{ github.sha }}",
-                "image":"${{ secrets.DOCKER_REGISTRY_URL || 'registry.vuestorefront.cloud' }}/${{ secrets.PROJECT_NAME }}-storefrontcloud-io/vue-storefront-middleware",
-                "path":"/api/",
-                "port":"3000",
-                "has_base_path":false
-              }],
-              "enabled": true
-              }
-            }' ${{ secrets.CLOUD_API_BASE_URL || 'https://farmer.vuestorefront.cloud' }}/instance/${{ secrets.PROJECT_NAME }}-${{ secrets.CLOUD_REGION }}-gcp-storefrontcloud-io; then
-            echo "Instance updated"
-          else
-            echo "Something went wrong during the update process..."
-            exit 1
-          fi
-
-      - name: Update deployment status (success)
-        if: success()
-        uses: chrnorm/deployment-status@releases/v1
-        with:
-          token: "${{ github.token }}"
-          target_url: https://${{ secrets.PROJECT_NAME }}.${{ secrets.CLOUD_REGION }}.gcp.storefrontcloud.io
-          state: "success"
-          description: Congratulations! The deploy is done.
-          deployment_id: ${{ steps.deployment.outputs.deployment_id }}
+    - name: Deploy
+      uses: fjogeleit/http-request-action@v1
+      with:
+        url: '${{ secrets.CLOUD_API_BASE_URL }}/cloud/instances/${{ secrets.PROJECT_NAME }}-${{ secrets.CLOUD_REGION }}-gcp-storefrontcloud-io/deploy'
```

Comment:
issue: Why it is the `CLOUD_API_BASE_URL` variable? It will point to Farmer API. I thought you said all requests should go through platform api?

Reply:
Excellent observation, you are completely correct.

Done: a94efad32eab750005e153d3bc954223758603aa